src/debugAdapter/goDebug.ts

```diff
@@ -271,7 +271,7 @@ class Delve {
 		} else if (typeof launchArgs['useApiV1'] === 'boolean') {
 			this.isApiV1 = launchArgs['useApiV1'];
 		}
-		this.stackTraceDepth = launchArgs.stackTraceDepth;
+		this.stackTraceDepth = launchArgs.stackTraceDepth || 50;
```

Comment:
I'm not sure where to place the constant since there are no other such constants in the file, should it be:
1. Somewhere at file scope? If so, where exactly?
2. Inside the class? (`private`/`public`?)
3. Alternatively, we can `import`/`require`/`JSON.parse` the `package.json` and get it from there. (`import`/`require` will require enabling the TypeScript setting `resolveJsonModule`)

@ramya-rao-a

Reply:
> It's also quite possible that there are other default values strawn around that aren't constant, 

Yes, right around this line of code we have set defaults for other properties of the `launchArgs` variable.

This looks good to me for now. I'll pick up the const issue in some later refactoring